dev/Prototypes/ItemsViewPrototype/FileExplorerScenario.xaml.cs

```diff
@@ -141,6 +191,7 @@ async void LoadItems()
             {
                 using (StreamReader sr = new StreamReader(stream))
                 {
+                    int index = 0;
```

Comment:
unused

Reply:
thanks for catching that. yes. I was experimenting with something there and forgot to remove